src/i18nResolver.ts

```diff
@@ -131,4 +132,4 @@ export class I18nResolver implements vscode.Disposable {
             this.fileSystemWatcher.dispose();
         }
     }
-}
```

Comment:
my editor is set up to add trailing newlines to files but I can clean this up if you don't want this

Reply:
Thats perfectly fine